apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -1,3 +1,5 @@
+import styles from "./button.module.css";
```

Comment:
We name all style files as styles.module.css
```suggestion
import styles from "./styles.module.css";
```